src/platform/packages/shared/kbn-esql-validation-autocomplete/src/definitions/generated/scalar_functions.ts

```diff
@@ -2757,6 +2757,7 @@ const kqlDefinition: FunctionDefinition = {
   examples: [
     'FROM books \n| WHERE KQL("author: Faulkner")\n| KEEP book_no, author \n| SORT book_no \n| LIMIT 5',
   ],
+  customSnippet: '"""$0"""',
```

Comment:
Maybe `customParametersSnippet` or something? Since you're only allowing to customize what is inserted between the parentheses (not a bad idea)

Reply:
Renamed in 84bc8d808820a8c37bc1739dff1c2bbba1d28476